systemd/README.md

```diff
@@ -0,0 +1,207 @@
+ExecStart=/home/livepeer/go-livepeer/livepeer -network rinkeby -transcoder -orchAddr 127.0.0.1:8935 -orchSecret osecret.txt -nvidia 0
+
+[Install]
+WantedBy=default.target
+```
+
+Update the `ExecStart` line to match your configuration. To use offline mode rather than the rinkeby testnet, replace the `ExecStart` line with the following:
+
+```
+ExecStart=/home/livepeer/go-livepeer/livepeer -network offchain -transcoder -orchAddr 127.0.0.1:8935 -orchSecret osecret.txt -nvidia 0
+```
+
+* Save the document and exit `vi` by first hitting the `ESC` key, then `:` followed by `wq` and `<ENTER>`
+
+* Enable and start the services:
+
+```bash
+systemctl enable transcoder.service
+systemctl start transcoder.service
+```
+
+* To monitor the service:
+
+```bash
+journalctl -f --unit=transcoder.service
+```
+
+To run the orchestrator and transcoder as a single combined service:
+
+* Create the `orchtran.service` file:
+
+```bash
+vi /etc/systemd/system/orchtran.service
+```
+
+* Push the `i` key to get into insertion mode.
+
+* Paste the following:
+
+```
+[Unit]
+Description=go-livepeer orchestrator+transcoder daemon service
+After=network.target
+
+[Service]
+User=livepeer
+Group=livepeer
+Type=simple
+Restart=always
+RestartSec=90s
+WorkingDirectory=/home/livepeer/go-livepeer/
+ExecStart=/home/livepeer/go-livepeer/livepeer -network rinkeby -orchestrator -transcoder -pricePerUnit 1 -nvidia 0 -initializeRound=true -serviceAddr=127.0.0.1:8935
+
+[Install]
+WantedBy=default.target
+```
+
+Update the `ExecStart` line to match your configuration. To use offline mode rather than the rinkeby testnet, replace the `ExecStart` line with the following:
+
+```
+ExecStart=/home/livepeer/go-livepeer/livepeer -network offchain -orchestrator -transcoder -serviceAddr 127.0.0.1:8935 -orchAddr 0.0.0.0:8935 -nvidia 0
+```
+
+* Save the document and exit `vi` by first hitting the `ESC` key, then `:` followed by `wq` and `<ENTER>`
+
+* Enable and start the services:
+
+```bash
+systemctl enable orchtran.service
+systemctl start orchtran.service
+```
+
+* To monitor the service:
+
+```bash
+journalctl -f --unit=orchtran.service
+```
+
+To run a broadcaster node service:
+
+* Create the `broadcaster.service` file:
+
+```bash
+vi /etc/systemd/system/broadcaster.service
+```
+
+* Push the `i` key to get into insertion mode.
+
+* Paste the following:
+
+```
+[Unit]
+Description=go-livepeer broadcaster daemon service
+After=network.target
+
+[Service]
+User=livepeer
+Group=livepeer
+Type=simple
+Restart=always
+RestartSec=90s
+WorkingDirectory=/home/livepeer/go-livepeer/
+ExecStart=/home/livepeer/go-livepeer/livepeer -network rinkeby -broadcaster -orchAddr 127.0.0.1:8935 -cliAddr 127.0.0.1:7936 -httpAddr 127.0.0.1:8936 -depositMultiplier 1
+
+[Install]
+WantedBy=default.target
+```
+
+Update the `ExecStart` line to match your configuration. To use offline mode rather than the rinkeby testnet, replace the `ExecStart` line with the following:
```

Comment:
same comment about offchain being the default for `-network` which allows us to omit the flag